requirement.txt

```diff
@@ -0,0 +1,26 @@
+amqp==1.4.6
```

Comment:
2 files are there for requirement

1. requirement.txt
2. requirements.txt

please keep the actual one.

also from requirement.txt are we using all the packages?